### ISSUE TITLE
Guest network not consistently displaying free and allocated addresses

### ISSUE THREAD

**Issue by adietrich-ussignal** [apache/cloudstack#9101]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI 
~~~

##### CLOUDSTACK VERSION

~~~
4.19.0.1
~~~

##### CONFIGURATION

Advanced Networking 
Shared Network w/ DHCP 

##### OS / ENVIRONMENT

Ubuntu 22.04 LTS 
Single-node Management Server 
MySQL 5.7

##### SUMMARY

When viewing the "IP Addresses" tab for a Guest Network (shared), the list of available IP addresses appears only part of the time. Using the refresh provided in the Guest Networks section, there is about a 60% success rate getting the information to correctly display guest IP addresses. 

##### STEPS TO REPRODUCE

~~~

1. Navigate to Network > Guest Networks
2. Select a Shared Network
3. Select the "IP Addresses" tab

~~~

##### EXPECTED RESULTS

~~~
A view of the allocated and free IP addresses within this network. 
~~~

![Screenshot 2024-05-21 at 4 03 44 PM](https://github.com/apache/cloudstack/assets/92819345/de3fb54b-83f0-4818-b6d6-9553bc99493a)

##### ACTUAL RESULTS

~~~
The screen returns a 'No Data' output. 

~~~
![Screenshot 2024-05-21 at 4 03 59 PM](https://github.com/apache/cloudstack/assets/92819345/9eba6c1d-f65a-4573-a4a6-de6ca3cdec83)

**Comment by shwstppr** [comment]
https://github.com/apache/cloudstack/pull/8624 will address this

**Comment by DaanHoogland** [comment]
fixed by #8624
